backend/fms_core/migrations/0054_v4_3_0.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 3.1 on 2023-07-12 18:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fms_core', '0053_v4_3_0'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='container',
+            index=models.Index(fields=['coordinate'], name='fms_core_co_coordin_27d459_idx'),
```

Comment:
I would recommend using human readable names for your indices... You can use a similar pattern than i use for unique keys (TableName_Field1_Field2_..._key) replacing key by idx for index... It will make it easier to understand traces and logs that might come up if the naming is clear (the default naming is truncated which fails to carry the information we want).